### ISSUE TITLE
There's no way to resolve relative `url`s

### ISSUE THREAD

**Issue by ainop** [docpad/docpad-plugin-stylus#16]
Hi! I keep my assets along with styl files and have similar structure in my styles directory:
```
src/documents/styles
├── styles.css.styl
├── page
│   └── page.styl
├── header
│   └── header.styl
├── logo
│   ├── logo.png
│   └── logo.styl
└── footer
    └── footer.styl
```
```
$ cat src/documents/styles/styles.css.styl
@import "*/*.styl"
```
When I build static, urls don't resolve. 'resolve url' in stylusOptions doesn't work as it's actually is a cli option and is handled in cli script: https://github.com/stylus/stylus/blob/961759bfcdcdcd437e62cbd6d3ab8bb6e5b4446e/bin/stylus#L797-L799.

I patched plugin to support 'resolve url', but not sure if it's legit way do such thing: https://github.com/ainop/docpad-plugin-stylus/commit/170a3cfe4cb51eb376a8c341b8079dee9f26b452

**Comment by kizu** [comment]
> I patched plugin to support 'resolve url', but not sure if it's legit way do such thing

Looks ok, feel free to submit a PR!

**Comment by kizu** [comment]
Closing as this was fixed in #17
